src/renderer/components/AccountInfo/AccountInfoContainer.tsx

```diff
@@ -84,6 +85,13 @@ const AccountInfoContainer: React.FC<Props> = (props: Props) => {
       currentReward.set(x!.itemId, x!.quantity);
     });
     setCurrentReward(currentReward);
+
+    const gold =
+      collectionStatus?.monsterCollectionStatusByAgent.fungibleAssetValue
+        .quantity ??
+      collectionStatusQuery?.monsterCollectionStatus?.fungibleAssetValue
+        .quantity;
+    setGold(Number(gold));
```

Comment:
This could be solved by useMemo too:

```typescript
const gold = useMemo(() =>
      collectionStatus?.monsterCollectionStatusByAgent.fungibleAssetValue
        .quantity ??
      collectionStatusQuery?.monsterCollectionStatus?.fungibleAssetValue
        .quantity, [collectionStatus, collectionStatusQuery]);

Reply:
i'll try this.